examples/MG_redshift_and_scale_dependences_examples.ipynb

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
This file should disappear from the repo before we merge.

Reply:
removed